Userland/Libraries/LibGL/SoftwareGLContext.h

```diff
@@ -161,10 +161,11 @@ class SoftwareGLContext : public GLContext {
     GLenum m_current_matrix_mode;
     FloatMatrix4x4 m_projection_matrix = FloatMatrix4x4::identity();
     FloatMatrix4x4 m_model_view_matrix = FloatMatrix4x4::identity();
-    FloatMatrix4x4 m_current_matrix = FloatMatrix4x4::identity();
+    FloatMatrix4x4 m_texture_matrix = FloatMatrix4x4::identity();
 
     Vector<FloatMatrix4x4> m_projection_matrix_stack;
     Vector<FloatMatrix4x4> m_model_view_matrix_stack;
+    Vector<FloatMatrix4x4> m_texture_matrix_stack;
```

Comment:
The texture matrix stack is actually part of the texture units and each unit has its own stack. `glActiveTexture` selects which is currently in use. https://www.khronos.org/registry/OpenGL/specs/gl/glspec15.pdf Page 43-44

Reply:
I'd say since we don't support multi texturing yet, adding a FIXME here would suffice.

Reply:
Agreed - I've added two `FIXME`s